_data/projects/flame-coach-web.yaml

```
@@ -0,0 +1,13 @@
+name: Flame Coach
+desc: Flame Coach, a simple way nutrition coaches track their client's progress! 
```

:art: no need to repeat the project name in the description

```suggestion
desc: A simple way nutrition coaches track their client's progress! 
```